azure-mgmt-network/azure/mgmt/network/v2017_06_01/models/load_balancer.py

```diff
@@ -110,3 +114,4 @@ def __init__(self, id=None, location=None, tags=None, frontend_ip_configurations
         self.resource_guid = resource_guid
         self.provisioning_state = provisioning_state
         self.etag = etag
+        self.zones = zones
```

Comment:
load balancer should not have zone. NRP folks have confirmed they have removed, please cross check the spec

Reply:
It was removed in 2017-09, but still in 2017-08 and 2017-06. I'm checking the spec.

Reply:
Ok, was removed from the spec, generation is not up to date. Fixing it.